### ISSUE TITLE
EFO:0000618 ‘nervous system disease’ has axiom to ‘central nervous system’ instead of ‘nervous system’

### ISSUE THREAD

**Issue by paolaroncaglia** [EBISPOT/efo#285]
Currently, EFO:0000618 ‘nervous system disease’ has_disease_location some ('central nervous system' or (part_of some 'central nervous system')). See https://www.ebi.ac.uk/ols/ontologies/efo/terms?iri=http%3A%2F%2Fwww.ebi.ac.uk%2Fefo%2FEFO_0000618. The axiom should point to 'nervous system' instead, see MONDO:0005071 ‘nervous system disorder’ (http://www.ontobee.org/ontology/MONDO?iri=http://purl.obolibrary.org/obo/MONDO_0005071). Note that this issue persists in EFO 3. Classification of children of EFO ‘nervous system disease’ should be looked at and refined where appropriate, because changing the axiom to make it more general will reduce the specificity of some of its children. At the same time, children of EFO 'peripheral nervous system disease' incorrectly map to 'central nervous system' instead of 'peripheral nervous system'.
The EFO term maps to the correct MONDO one in EFO 3, but the axiom to the UBERON term wasn't updated.

**Comment by zoependlington** [comment]
- [ ] I will try and pull out this term and all descendant terms from EFO 2 into a spreadsheet so we can look at each term individually and compare against MONDO.

**Comment by paolaroncaglia** [comment]
@zoependlington 
I'm assigning this ticket to you as you kindly offered to address the first step (when you can). I can then take on the rest. Thanks.

**Comment by zoependlington** [comment]
Here are all descendants of nervous system disease: https://docs.google.com/spreadsheets/d/1WXyG2T8Doaj6nAijEa3m2sqBbemCdaKct9827YrgrYM/edit?usp=sharing

**Comment by paolaroncaglia** [comment]
@zoependlington 
Thanks for doing this! I took a look and, if you don't mind and when you have time, it would help if you could please:
- [ ] produce a spreadsheet like the one above, but with only the direct children of EFO 'nervous system disease' (i.e. not all descendants), to make things simpler;
- [ ] produce a similar spreadsheet for direct children of MONDO:0002602 'central nervous system disease';
- [ ] produce a similar spreadsheet for direct children of MONDO:0003620 'peripheral nervous system disease'.
Or 3 tabs in the same spreadsheet, if that's quicker.
As I said, no hurry, and many thanks in advance!

**Comment by zoependlington** [comment]
All done @paolaroncaglia! I added them as tabs to the same spreadsheet. Let me know if there's anything else I can help with! 🙂

**Comment by paolaroncaglia** [comment]
Thanks @zoependlington !

**Comment by paolaroncaglia** [comment]
Notes for self.
In the fifth tab of the spreadsheet (https://docs.google.com/spreadsheets/d/1WXyG2T8Doaj6nAijEa3m2sqBbemCdaKct9827YrgrYM/edit#gid=2113892663), I compared the direct children of EFO 'nervous system disease' with the direct children of MONDO 'central nervous system disease' and MONDO 'peripheral nervous system disease'. First thoughts:

The exact overlaps are few and marked in colours. I also added comments to a couple of terms (visible as orangey corners in the column of EFO labels).

The hierarchy in EFO is flatter than necessary. This is probably because terms with granular location axioms also keep the direct Subclass of 'nervous system disease'. Quite a bit of work needed on axioms. Fix the axioms first for 'nervous system disease', 'central nervous system disease' and 'peripheral nervous system disease', run the reasoner, classify terms in identical color where necessary, then go from there.

**Comment by paolaroncaglia** [comment]
- [ ] Add EFO:NEW 'central nervous system disease' [new ID: EFO:0009386]
Subclass of 'nervous system disease'
Def "A disease involving the central nervous system."
definition_citation MONDO:0002602
synonym: central nervous system disorder

**Comment by paolaroncaglia** [comment]
Notes for self:

1) We agreed that we should apply the following pattern for disease location, making sure that the brackets are the same across all equivalence axioms of the same type:

equivalentTo:
text: "%s and (has_disease_location some (%s or ('part of' some %s)))"
vars:
- disease
- location
- location

2) There is indeed an issue with UBERON import in EFO. When that's fixed, it'll help with this ticket too, so perhaps wait on https://github.com/EBISPOT/efo/issues/310 to be done.

**Comment by paolaroncaglia** [comment]
Note for self:
Now that the Uberon import has been completed and EFO3 is the default, I'll move this to 'Next'. 
I can then assess if/how badly the ‘nervous system disease’ branch needs work, and prioritize such work based on its relevance for our stakeholders.

**Comment by paolaroncaglia** [comment]
Agreed that this can stay in Next. If it can't be done in the coming sprint, that's fine.

**Comment by paolaroncaglia** [comment]
Tagging this for potential proposal. May ignore the full length of the ticket and focus on the following:
Review EFO-only children of 'nervous system disease' and mark if they could/should be placed more specifcally under 'central nervous system disease', 'peripheral nervous system disease', or any other high-level term under 'nervous system disease'.
